Vostok.Metrics.System/Host/DiskUsageCollector_Linux.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Vostok.Metrics.System.Helpers;
+
+namespace Vostok.Metrics.System.Host
+{
+    internal class DiskUsageCollector_Linux
+    {
+        private Dictionary<string, DiskStats> previousDisksStatsInfo = new Dictionary<string, DiskStats>();
+        private readonly Stopwatch stopwatch = new Stopwatch();
+
+        public void Collect(HostMetrics metrics, IEnumerable<string> diskstatsLines)
+        {
+            var deltaTime = (double) stopwatch.ElapsedTicks / Stopwatch.Frequency;
+
+            var disksUsageInfo = new Dictionary<string, DiskUsageInfo>();
+
+            foreach (var diskStats in ParseDiskstats(diskstatsLines))
+            {
+                var result = new DiskUsageInfo {DiskName = diskStats.DiskName};
+
+                if (previousDisksStatsInfo.ContainsKey(diskStats.DiskName))
```

Comment:
Not that it's of any relevance here, but it's a good habit to use `TryGetValue` to avoid two lookups in the dictionary (one for `ContainsKey` and one for the indexer access). Might come in handy one day in perfomance-sensitive code.